dask_match/core.py

```diff
@@ -307,20 +313,32 @@ def _divisions(self):
     def _name(self):
         return funcname(self.operation) + "-" + tokenize(*self.operands)
 
-    def _layer(self):
+    def _subgraph_dependencies(self):
+        # List `Expr` operands only.
+        # This property is required to enable fusion
+        return [operand for operand in self.operands if isinstance(operand, Expr)]
+
+    def _block_subgraph(self):
```

Comment:
Slight preference to name this either `_blockwise_subgraph` or `_subgraph` to either reuse the existing term (blockwise) or keep things short.